.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
     - NOSE_VERBOSE=2
     - NOSE_WITH_COVERAGE=true
     - NOSE_COVER_PACKAGE=neovim
```

Comment:
we don't use nose anymore, so we can remove these envs I guess?

Reply:
Likely, yes.  Should be a seperate PR then though.